test/build.info

```diff
@@ -859,9 +859,9 @@ IF[{- !$disabled{tests} -}]
   DEPEND[namemap_internal_test]=../libcrypto.a libtestutil.a
 
   PROGRAMS{noinst}=bio_prefix_text
-  SOURCE[bio_prefix_text]=bio_prefix_text.c $LIBAPPSSRC
+  SOURCE[bio_prefix_text]=bio_prefix_text.c
   INCLUDE[bio_prefix_text]=.. ../include ../apps/include
-  DEPEND[bio_prefix_text]=../libcrypto
+  DEPEND[bio_prefix_text]=../libcrypto libtestutil.a
```

Comment:
As I recall, there was a reason to avoid libtestutil.a with this program

Reply:
Side note: I hadn't noticed it before; isn't there something a bit odd with the name bio_prefix_te**x**t

Reply:
Can you elucidate?

The commit message is content free, there are no comments of note in the source file and the PR commentary mentions nothing.

Reply:
The test is about indentation of test and prefixing with test as part of a BIO, so the name seems reasonable.

Reply:
I can't elucidate for the moment. I'm running a build on VMS to see if that's where I saw trouble....

Reply:
Re the name, I see it as a test of bio_prefix.  Can't recall having wanted to make a filename distinction re te**x**t.  I rather think that when I made up the name, I fumbled, with my fingers writing "text" although I wanted "test".

Reply:
I make that type a lot too.